### PR TITLE
update to bartholomew v0.8.0 and remove common folder

### PR DESCRIPTION
**Pull request by karthik2804**
Updates to `bartholomew v0.8.0` and uses dynamic body source to share content and thus removing the need to have common documents under a common folder.

### CODE REVIEW

**Comment by itowlson** [comment]
Does this remove the common folder @karthik2804?  GH is showing me only that the Wasm file has changed but I'm not sure if that's just a UI glitch on deletions or something.

**Comment by itowlson** [comment]
(I mean no worries the update is fine by itself, just making sure of your intention.)

**Comment by karthik2804** [comment]
oh @itowlson, I just completely glanced over the fact that I did not check in all the changes 🤷‍♂️ .  I have fixed it now. My apologies for the blunder.

**Review by itowlson — APPROVED** [review]
🎉 :shipit:

File: spin.toml [hunk 1]
```diff
@@ -215,3 +170,13 @@ environment = { DESTINATION = "https://developer.fermyon.com/spin/kv-store-tutor
...
+
+
+# Redirecting from common to spin incase someone had it bookmarked before removal of common
+[[component]]
+id = "redirect-contributing-common-to-spin"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/cli-reference" }
+[component.trigger]
+route = "/common/cli-reference"
+executor = { type = "wagi" }
```
Comment (karthik2804) [inline spin.toml:182]: @itowlson Do you think we need to keep this around or is this something we can remove?

Comment (itowlson) [reply spin.toml:182]: Leave it in case there are any dangling links to it

**Pull request merged**